deploy/vagrant/core/config_aws.rb

```diff
@@ -25,6 +24,13 @@ def config_aws(config, i, total, name)
       'Name' => TAG + "-" + name,
     }
 	  aws.availability_zone = AVAILABILITY_ZONE
+    if (SUBNET != nil and SUBNET != "")
+      aws.subnet_id = SUBNET
+      aws.associate_public_ip = TRUE
+      aws.security_groups = ENV['AWS_SECURITY_GROUP_ID_TACH']
```

Comment:
do you mean the environment variable?  I can probably add a check to make sure it is not an empty string.